### ISSUE TITLE
Multitenant: ApiUsers resource is broken

### ISSUE THREAD

**Issue by julsemaan** [inverse-inc/packetfence#2813]
First when trying it out:
```
Dec 04 12:46:18 bouzinfence pfunified_api[21847]: [Mon Dec  4 12:46:18 2017] [error] Can't locate object method "new" via package "pf::dal::api_user" (perhaps you forgot to load "pf::dal::api_user"?) at /usr/local/pf
```

Then also, in the ApiUsers controller, it looks like a copy/paste of the Users API controller where its looking for the PID as the ID column, but that table uses username instead.

Please make sure that the ApiUsers + Users controller work and add some tests for them for CRUD operations and validations that the DB records are right when calls are made through the API

**Comment by jrouzierinverse** [comment]
ok

**Comment by jrouzierinverse** [comment]
Fixed by 09231c125db21987db991bb6f717148fa91a7859
